web/src/app/converters/proto-model-converter.ts

```diff
@@ -225,3 +210,25 @@ function taskConditionToPartialMessage(task: Task): Pb.Task.ICondition[] {
       .toArray() || []
   );
 }
+
+/**
+ * Returns the proto representation of a list of Task model objects.
+ */
+export function tasksToDocument(tasks: List<Task>): Pb.ITask[] {
```

Comment:
These methods return proto Messages, not Firestore Documents. Please rename to `*ToMessage` where appropriate.